arrows/core/metadata_map_io_csv.cxx

```diff
@@ -59,15 +59,17 @@ class metadata_map_io_csv::priv
 };
 
 namespace {
+
 // ----------------------------------------------------------------------------
 struct write_visitor {
-  static constexpr auto crs = kwiver::vital::SRID::lat_lon_WGS84;
+  static constexpr auto lat_lon_WGS84 = kwiver::vital::SRID::lat_lon_WGS84;
 
   template< class T >
   void
   operator()( T const& data ) const
   {
-    os << data << ",";
+    // TODO: handle pathalogical characters such as quotes or newlines
+    os << "\"" << data << "\",";
```

Comment:
I think this is better for "unknown" types (and consistency with the old code), but maybe we should add an overload for numeric types that does not quote?
```suggestion
    if( std::is_arithmetic< T > )
    {
      os << data << ',';
    }
    else
    {
      // TODO: handle pathalogical characters such as quotes or newlines
      os << "\"" << data << "\",";
    }
```

https://en.cppreference.com/w/cpp/types/is_arithmetic